app/src/main/java/com/duckduckgo/app/email/EmailJavascriptInterface.kt

```diff
@@ -44,8 +44,8 @@ class EmailJavascriptInterface(
     fun isSignedIn(): String = emailManager.isSignedIn().toString()
 
     @JavascriptInterface
-    fun storeCredentials(token: String, username: String) {
-        emailManager.storeCredentials(token, username)
+    fun storeCredentials(token: String, username: String, cohort: String) {
```

Comment:
Just checking if we are sure that `cohort` will never be `null`.

Reply:
Yup, already confirmed by the FE team.